react-ui-kit/src/Select/components/Others.tsx

```diff
@@ -33,8 +45,11 @@ export const SelectContainerComponent: FC<any> = (props) => {
         </Label>
       )}
       <SelectContainer>
-        <components.SelectContainer {...{...props, className: ""}}>{children}</components.SelectContainer>
+        <components.SelectContainer {...{ ...props, className: '' }}>
```

Comment:
```suggestion
        <components.SelectContainer {...props} className="">
```
Должно работать также, но читать легче

Reply:
тоже поправила